content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -130,6 +130,23 @@ defined range of integers representing the minimum and maximum scores. After the
 [NormalizeScore](#normalize-scoring) phase, the scheduler will combine node
 scores from all plugins according to the configured plugin weights.
 
+#### Capacity scoring {#scoring-capacity}
+
+{{< feature-state feature_gate_name="StorageCapacityScoring" >}}
+
+This feature adds a scoring method for pod scheduling with
+[the topology-aware volume provisioning](/blog/2018/10/11/topology-aware-volume-provisioning-in-kubernetes/).
+This feature eases to schedule pods on nodes with either the most or least
+available storage capacity. This feature originally extended the
```

Comment:
? "extended" what?

Reply:
Originally, `StorageCapacityScoring` was a feature that added dynamic provisioning support to `VolumeCapacityPriority`, which only supported static provisioning. However, because `VolumeCapacityPriority` was going to be removed, `StorageCapacityScoring` became the feature gate that includes both static and dynamic provisioning.

Would "added" be more appropriate than "extended" in this context?

Reply:
I have modified this sentence as follows.

> Originally, `StorageCapacityScoring` was a feature that added dynamic provisioning support to `VolumeCapacityPriority`, which only supported static provisioning. However, because `VolumeCapacityPriority` was going to be removed, `StorageCapacityScoring` became the feature gate that includes both static and dynamic provisioning.

Reply:
How about this?
> The feature gate `VolumeCapacityPriority` was used in v1.32 to support storage that are
> statically provisioned. Starting from v1.33, the new feature gate `StorageCapacityScoring`
> replaces the old `VolumeCapacityPriority` gate with added support to dynamically provisioned storage.
> When `VolumeCapacityScoring` is enabled, the VolumeBinding plugin in the kube-scheduler is extended
> to score Nodes based on the storage capacity on each of them.
> This feature is applicable to all CSI volumes, including local storage backed by
> a CSI driver.

Reply:
Thank you very much!

> When `VolumeCapacityScoring` is enabled,

(I think this is just a typo) It should be `StorageCapacityScoring` now.

> This feature is applicable to all CSI volumes,

The phrase "all CSI volumes" could be a bit misleading. This feature requires storage capacity information obtained from [Storage Capacity](https://kubernetes.io/docs/concepts/storage/storage-capacity/). Therefore, CSI volumes that do not support [Storage Capacity](https://kubernetes.io/docs/concepts/storage/storage-capacity/) are not covered by this feature.

Nevertheless, thank you. I will try to revise the text based on your comments.

Reply:
I addressed it.